server/src/main/java/dev/slimevr/tracking/trackers/udp/TrackersUDPServer.kt

```diff
@@ -293,7 +293,8 @@ class TrackersUDPServer(private val port: Int, name: String, private val tracker
 			is UDPPacket4Acceleration -> {
 				tracker = connection?.getTracker(packet.sensorId)
 				if (tracker == null) return
-				val acceleration = tracker.getRotation().sandwich(packet.acceleration)
+				val accelRot = tracker.resetsHandler.getMountingAdjustedRotationFrom(tracker.getRawRotation())
+				val acceleration = accelRot.sandwich(packet.acceleration)
```

Comment:
You would want to do that directly in TrackerResetsHandler, as trackers with accel can be created from other sources

Reply:
good point let me know if the changes I made are what you had in mind

Reply:
This approach still works obviously, but we risk someone forgetting to call that method when setting accel somewhere else, and it also separates the logic from where I think it should be.  

What I had in mind was doing the same thing as rotation (in Tracker.kt, make accel private and have a custom get method that gets the adjustedAccelerationFromRotation if tracker needsReset).

Reply:
that makes more sense, the naming of needs reset threw me off a little I thought it indicated something else, thanks for elaborating